code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPConfig.java

```diff
@@ -164,7 +168,7 @@ public String getGroupNameField() {
         return groupNameField;
     }
 
-    @Field(nullable = false, required = true, defaultValue = "gidNumber")
+    @Field(nullable = false, required = true, defaultValue = "memberOf")
```

Comment:
I assume gidNumber is still supported as a valid argument? Just making sure that existing setups won't break